lib/stormpath-sdk/resource/application.rb

```diff
@@ -117,6 +118,10 @@ def get_provider_account request
   def oauth_authenticate(request)
     Stormpath::Oauth::Authenticator.new(data_store).authenticate(href, request) 
   end
+  
+  def create_login_attempt(options = {})
```

Comment:
What I'm trying to say is that `login_attempts` shouldn't be necessary at all. This is because the existing `authenticate_account` method should be able to handle the login attempt without adding any new properties or classes.

So, people should be able to do the following:

```ruby
auth_request  = Stormpath::Authentication::UsernamePasswordRequest.new (
'johnsmith', 
'4P@$$w0rd!', 
{accountStore: "accountStore Href or nameKey"})

auth_result = application.authenticate_account auth_request
```